tests/integration_tests/functional/test_balloon.py

```diff
@@ -298,6 +298,16 @@ def test_reinflate_balloon(test_microvm_with_ssh_and_balloon, network_config):
     firecracker_pid = test_microvm.jailer_clone_pid
     ssh_connection = net_tools.SSHConnection(test_microvm.ssh_config)
 
+    # First inflate the balloon to free up the uncertain amount of memory
+    # used by the kernel at boot and establish a baseline, then give back
+    # the memory.
+    response = test_microvm.balloon.patch(amount_mb=200)
+    assert test_microvm.api_session.is_status_no_content(response.status_code)
+    time.sleep(5)
```

Comment:
After the offline discussion that we had, I think these wait times need to be addressed in a generic way. As you said in the corresponding issue, we need a generic mechanism for Firecracker process RSS monitoring. Should we track this separately in a new issue?

Reply:
Yes, I'll log a separate issue as soon as this is merged.